config/config.yaml

```diff
@@ -115,9 +115,7 @@ pimcore:
 #### Pimcore ADMIN
 pimcore_admin:
     admin_csp_header:
-        enabled: false
-        exclude_paths:
-            - '@^/admin/external_opcache@'
```

Comment:
we should opcache exclude path if system info is enabled on demo

Reply:
System info is extracted to it's own repo and not enabled on demo at the moment. So, what about updating this in the ReadMe/documentation of system-info bundle